### PR TITLE
[12.0][MIG] sale_stock_secondary_unit: Migration to 12.0

### PR DESCRIPTION
**Pull request by digitalsatori**

### CODE REVIEW

File: sale_stock_secondary_unit/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     'name': 'Sale Stock Secondary Unit',
     'summary': 'Get product quantities in a secondary unit',
-    'version': '11.0.1.0.0',
+    'version': '12.0.1.0.0',
     'development_status': 'Beta',
     'category': 'stock',
     'website': 'https://github.com/OCA/stock-logistics-warehouse',
```
Comment (nikul-serpentcs) [inline sale_stock_secondary_unit/__manifest__.py]: @digitalsatori URL like `https://github.com/OCA/sale-workflow`

Comment (digitalsatori) [reply sale_stock_secondary_unit/__manifest__.py]: Thank you @nikul-serpentcs , changed in the new commit.

**Comment by sergio-teruel** [comment]
travis red due to sale_automatic_workflow. @pedrobaeza this test was fixed?
Functional test OK

**Review by sergio-teruel — APPROVED** [review]
Waiting for green colors :smile:

**Comment by pedrobaeza** [comment]
I suppose the PR is not rebased, but the bot can tell us, so /ocabot merge

**Comment by sergio-teruel** [comment]
@pedrobaeza this continues to fail...

**Comment by pedrobaeza** [comment]
The problem is the hour when is around midnight, due to UTC/current zone deviation. Tests should be resilient to this, but meanwhile, doing the test for example at this hour should be enough.

/ocabot merge

**Comment by digitalsatori** [comment]
wow, that's great to know. Thank you folks for the great effort and your perseverance.

**Pull request merged**